### PR TITLE
Style Guide Updates to Training-Data-Format Docs

### PR DESCRIPTION
**Pull request by b-quachtran**
**Proposed changes**:
- ...

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by b-quachtran** [comment]
@yennycheung PR with the edits from earlier today.

File: docs/docs/training-data-format.mdx [hunk 1]
```diff
@@ -34,21 +33,20 @@ you can still find the documentation for
...
-All YAML training data files should specify the `version` key to be parsed
-correctly. If you don't specify a version key in your training data file, Rasa
+You should specify the `version` key in all YAML training data files.
+If you don't specify a version key in your training data file, Rasa
 will assume you are using the latest training data format specification supported
 by the version of Rasa Open Source you have installed.
-Training data files with a version greater than is currently available in
-the version of Rasa Open Source you have installed will be
-skipped.
+Training data files with a Rasa Open Source version greater than what you have
```
Comment (erohmensing) [inline docs/docs/training-data-format.mdx:48]:
```suggestion
Training data files with a Rasa Open Source version greater than the version you have
```

File: docs/docs/training-data-format.mdx [hunk 2]
```diff
@@ -100,7 +98,7 @@ stories:
     intent: faq/language
   - action: utter_faq
 ```
-They use the same format as the story training data.
+Test stories use the same format as the story training data.
```
Comment (erohmensing) [inline docs/docs/training-data-format.mdx:101]: L89: link to test stories section, and be expicit that the name of the separate file needs to start with `test_

Comment (erohmensing) [reply docs/docs/training-data-format.mdx:101]: also "If you want to" -> "To"

File: docs/docs/training-data-format.mdx [hunk 3]
```diff
@@ -673,7 +673,7 @@ stories:
   - action: utter_ask_price
 ```
 
-Running the tests can be done using the CLI:
+You can run the tests using the following command:
```
Comment (erohmensing) [inline docs/docs/training-data-format.mdx:676]: Please update anywhere in this section that refers to `test conversations` to `test stories`

**Review by erohmensing — APPROVED** [review]

**Comment by erohmensing** [comment]
Please rebase to 2.0.x before merging!

**Comment by erohmensing** [comment]
Removing label, as it doesn't seem the rebase worked 😕

**Comment by b-quachtran** [comment]
Hmm, I'll take a look at what's going on. Rebasing nightmares are the best 🤦

**Comment by b-quachtran** [comment]
@erohmensing Eh, I'm just gonna create a new PR off of 2.0.x with this file change

**Pull request closed without merging**